### ISSUE TITLE
Unable to delete form with very long handle

### ISSUE THREAD

**Issue by mtnorthrop** [verbb/formie#360]
**Description**

When a form is duplicated, a string of numbers (timestamp?) is added to the end of the original form's handle. I've just had a case where a client duplicated a form with a long name a couple of times, and then was unable to delete the copy. These are the errors that were shown in `web.log`:

```
PDOException: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '{{fmc_prihlaskanajazykovykurz2517552231377420010275965341567969'

Next yii\db\Exception: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '{{fmc_prihlaskanajazykovykurz2517552231377420010275965341567969' at line 1

The SQL being executed was: SHOW FULL COLUMNS FROM {{fmc_prihlaskanajazykovykurz2517552231377420010275965341567969
```

The string with the handle that was quoted in the errors above is actually a truncated form of the form's actual handle. After changing the handle to something shorter, I was able to successfully delete the form.

The form's original handle was `fmc_prihlaskanajazykovykurz2517552231377420010275965341567969470`, which is exactly 64 chars. The SQL quoted in the error messages is truncating the last 3 chars.

My guess is that this was caused by the handle being truncated somewhere in the database. I believe the character limit for tables in the database is 64 chars, and I see that the `handle` column length in the `formie_forms` table is 64 chars as well, so I'm guessing the handle got truncated somewhere.

**Additional info**
- Plugin version: dev-craft-3 as 1.3.13
- Craft version: 3.6.6
- Multi-site: Yes

**Additional context**

**Comment by engram-design** [comment]
Interesting, you shouldn't be able to create a form with a handle longer than 60 characters in the first place. I've fixed a number of issues that are the source of this issue, so it (hopefully) shouldn't happen again. Namely, better new form handle generation, fixing being able to save a form with a longer-than 60 character (when saving as a new form), and better error feedback in the CP.

To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.18",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.3.19](https://github.com/verbb/formie/releases/tag/1.3.19)
